skll/config.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
                                    'min_feature_count': 'Tuning',
                                    'models': 'Output',
                                    'num_cv_folds': 'Input',
-                                   'objective': 'Tuning',
```

Comment:
I think that's why I was saying that it should stay in. For backward compatibility, we are still accepting that option, but we should also check that it is in the correct section?